readme_renderer/clean.py

```diff
@@ -59,15 +60,12 @@
     "input": ["type", "checked", "disabled"],
 }
 
-ALLOWED_STYLES = [
-]
-
 
 class DisabledCheckboxInputsFilter:
-    def __init__(self, source):
+    def __init__(self, source: typing.Any) -> None:
```

Comment:
I'm a little surprised to see this use of `Any`, given the `--strict` configuration. Is there a way to make it a specific type, or a `Union`?

Reply:
Strict mode doesn't disallow any Any (pun!), rather a subset of all strictness flags.

> Strict mode; enables the following flags: --warn-unused-configs, --disallow-any-generics, --disallow-subclassing-any, --disallow-untyped-calls, --disallow-untyped-defs, --disallow-incomplete-defs, --check-untyped-defs, --disallow-untyped-decorators, --no-implicit-optional, --warn-redundant-casts, --warn-unused-ignores, --warn-return-any, --no-implicit-reexport, --strict-equality

I'll take another look at the potential union for this, and a couple others, but some had to do with not being a subclass with typing.

Reply:
No union type available, upstream types are incomplete, so will add a comment for now.